nais/naiserator.yml

```diff
@@ -29,6 +29,9 @@ spec:
   {{#each ingresses as |url|}}
      - {{url}}
   {{/each}}
+  outbound:
+    external:
+      - host: api.dips.no
```

Comment:
Treng vi egentlig dette pr no? Såvidt eg veit utføre vi ikkje noko serverside kall til api.dips.no (endå)

Reply:
Bra spotta! 
Det er riktig, vi trenger ikke å åpne for eksterne hosts når vi kjører klientside 👍